app/src/main/java/com/github/freeman/bootcamp/MainMenuActivity.kt

```diff
@@ -285,7 +287,9 @@ fun MainMenuScreen() {
                 SettingsButton()
                 ChatButton()
             }
-            CreateVideoButton()
+            Row {
+                DisplayRecapsButton()
+            }
```

Comment:
It's a detail and isn't wrong, but why put the button in a `Row{}` if it's the only element in it ?

Reply:
It was actually because I used multiple buttons but it was for testing purpose and it's not used anymore. I will remove the row then